repository/Seaside-Core.package/WAUrl.class/class/readHexFrom.errorDetail..st

```diff
@@ -0,0 +1,10 @@
+private
+readHexFrom: aStream errorDetail: aString
+	| hexChar |
+	aStream atEnd ifTrue: [
+		WAInvalidUrlSyntaxError signal: aString ].
+	hexChar := aStream next.
```

Comment:
Would bit shift operations make sense here to improve performance?

Reply:
The temptation to optimize is there but I think it's unlikely this shows up during profiling.